src/Packagist/WebBundle/Resources/public/css/main.css

```diff
@@ -1003,10 +1003,10 @@ ul.packages .metadata-block:last-child {
 .package-aside .details {
   padding-left: 0;
 }
-.package .package-aside i, .package .readme-expander i {
+.package .package-aside i:not(.fa), .package .readme-expander i:not(.fa) {
```

Comment:
well, these CSS rules are explicitly excluding your FA icons, which is why I suggest making it target the other icons explicitly instead